src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_INameOfOperation.cs

```diff
@@ -0,0 +1,178 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
+{
+    public partial class IOperationTests : SemanticModelTestBase
+    {
+        [CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Dataflow)]
+        [Fact]
+        public void NameOfFlow_01()
+        {
+            string source = @"
+class C
+{
+    void M(bool b, int i1, int i2)
+    /*<bind>*/{
+        string test = nameof(b ? i1 : i2);
```

Comment:
>string test [](start = 8, length = 11)

In the future CFG PRs, consider avoiding declaring locals unless you are specifically testing an aspect related to declaring locals. The reason is that creates regions in the graph and somewhat bloats base-lines, creates more stuff to process for a human.